### ISSUE TITLE
While rebasing to master I get the following exeption from the plubing

### ISSUE THREAD

**Issue by csitarichie** [ArtsiomCh/CMake#8]
It happened only ones ... don't really know how to reproduce it :). Sorry 

But I was thinking it is worth to report it :).
```
Should be called at least in the state COMPONENTS_LOADED, the current state is: CONFIGURATION_STORE_INITIALIZED
Current violators count: 1



java.lang.Throwable
	at com.intellij.diagnostic.LoadingState.checkOccurred(LoadingState.java:74)
	at com.intellij.psi.tree.IElementType.<init>(IElementType.java:91)
	at com.jetbrains.cmake.psi.CMakeElementType.<init>(CMakeElementType.java:9)
	at com.jetbrains.cmake.psi.CMakeTokenTypes.<clinit>(CMakeTokenTypes.java:10)
	at com.jetbrains.cmake.psi.CMakeElementTypes.<clinit>(CMakeElementTypes.java:10)
	at com.cmakeplugin.utils.CMakeProxyToJB.<clinit>(CMakeProxyToJB.java:80)
	at com.cmakeplugin.utils.CMakePDC.<clinit>(CMakePDC.java:18)
	at com.cmakeplugin.CMakeFileTypeFactory.createFileTypes(CMakeFileTypeFactory.java:10)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.lambda$initStandardFileTypes$0(FileTypeManagerImpl.java:363)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processWithPluginDescriptor(ExtensionPointImpl.java:311)
	at com.intellij.openapi.extensions.ExtensionPointName.processWithPluginDescriptor(ExtensionPointName.java:157)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.initStandardFileTypes(FileTypeManagerImpl.java:361)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.<init>(FileTypeManagerImpl.java:230)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:44)
	at com.intellij.serviceContainer.PlatformComponentManagerImpl.instantiateClassWithConstructorInjection(PlatformComponentManagerImpl.kt:500)
	at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:52)
	at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceComponentAdapter.kt:39)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:110)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:72)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance$default(BaseComponentAdapter.kt:65)
	at com.intellij.serviceContainer.PlatformComponentManagerImpl$preloadServices$future$1.run(PlatformComponentManagerImpl.kt:612)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:222)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:30)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:201)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:193)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by ArtsiomCh** [comment]
Yep, saw it in the log (shouldn't be shown to the user actually). Just ignore it for now. All plugin functionality works.

**Comment by ArtsiomCh** [comment]
Fixed at 193.0.3
